core/query/assets.go

```diff
@@ -29,6 +30,12 @@ func (ind *Indexer) SaveAnnotatedAsset(ctx context.Context, asset *AnnotatedAsse
 	return errors.Wrap(err, "saving annotated asset")
 }
 
+func (ind *Indexer) UpdateAnnotatedAssetTags(ctx context.Context, id bc.AssetID, tags json.RawMessage) error {
+	const q = "UPDATE annotated_assets SET tags = $1::jsonb WHERE id = $2"
+	_, err := ind.db.Exec(ctx, q, string(tags), id)
+	return errors.Wrap(err, "update query")
+}
```

Comment:
Should be able to update the `assets` table, call `Annotated(a)` to get the annotated asset and then save it with `SaveAnnotatedAsset`.